jplag.cli/src/main/java/de/jplag/logger/CollectedLogger.java

```diff
@@ -0,0 +1,270 @@
+package de.jplag.logger;
+
+import java.io.PrintStream;
+import java.io.Serial;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.concurrent.ConcurrentLinkedDeque;
+
+import org.slf4j.helpers.FormattingTuple;
+import org.slf4j.helpers.MarkerIgnoringBase;
+import org.slf4j.helpers.MessageFormatter;
+import org.slf4j.spi.LocationAwareLogger;
+
+/**
+ * This logger is able to collect errors and print them at the end. Mainly adopted from org.slf4j.impl.SimpleLogger
+ * @author Dominik Fuchss
+ */
+public final class CollectedLogger extends MarkerIgnoringBase {
```

Comment:
Quite a few abbreviations in identifiers in this class.

Reply:
Yes, but as I've written above. The code is mainly adopted from the SimpleLogger. Therefore, I didn't changed any naming (just added a few lines of code and removed some option-related code lines :) )

Esp. I'm not sure whether some names are important or not (I guess not, but I'm not sure about it :) )

Reply:
I think we should still adapt it to our naming conventions as for future maintenance nobody will know this reasoning. Furthermore, applying some renaming refactorings is not the hardest thing to do ;)

Reply:
Ok. I'll work on it asap

Reply:
Done